docs/framework/tools/al-exe-assembly-linker.md

```diff
@@ -32,8 +32,8 @@ You can specify one or more of the following `sources`.
 | Source | Description |
 | ------ | ----------- |
 |`file`[,`target`]|Copies the contents of `file` (a module) to the file name specified by `target`. After copying, *Al.exe* compiles `target` into an assembly.|
-|**/embed[resource]:** `file`[,`name`[,`private`]]|Embeds the resource specified by `file` in the image that contains the assembly manifest; *Al.exe* copies the contents of `file` into the portable executable (PE) image.<br /><br /> The `name` parameter is an internal identifier for the resource. By default, resources are public in the assembly (visible to other assemblies). Specifying `private` makes the resource not visible to other assemblies.<br /><br /> If `file` is a .NET Framework resource file created, for example, by the [Resource File Generator (*Resgen.exe*)](resgen-exe-resource-file-generator.md) or in the development environment, it can be accessed with members in the <xref:System.Resources>. For more information, see <xref:System.Resources.ResourceManager>. For all other resources, use the `GetManifestResource`* methods in the <xref:System.Reflection.Assembly> to access the resource at run time.<br /><br /> If only resource files are passed to *Al.exe*, the output file is a satellite resource assembly.|
```

Comment:
Two instances of the same edit here: `GetManifestResource`* -> `GetManifestResource*`